format/index.md

```diff
@@ -1758,12 +1760,24 @@ An error object **MAY** have the following members:
     the error.
 * `meta` - to contain non-standard meta-information about the error.
 
+[top level]: #document-structure-top-level
 [attributes]: #document-structure-resource-attributes
+[attributes object]: #document-structure-resource-attributes
 [relationships]: #document-structure-resource-objects-relationships
+[relationships object]: #document-structure-resource-objects-relationships
 [resource relationships]: #document-structure-resource-objects-relationships
+[relationship object]: #document-structure-resource-objects-relationship-objects
```

Comment:
no need to have multiple entries for the same url. you can do [anything at all here][actual name]